lib/ansible/modules/cloud/vmware/vmware_host_firewall_manager.py

```diff
@@ -173,8 +173,19 @@
 
 from ansible.module_utils.basic import AnsibleModule
 from ansible.module_utils.vmware import vmware_argument_spec, PyVmomi
-from ansible.module_utils._text import to_native, to_text
-from ansible.module_utils.compat import ipaddress
+from ansible.module_utils._text import to_native
+import socket
+
+
+def is_ipaddress(value):
```

Comment:
do you think we should move this function to module_utils/vmware.py?

Reply:
I think we can continue this way. So far, it's the only place where we use it.